console/line.go

```diff
@@ -339,7 +334,11 @@ func (c *Component) graphLineHandlerFunc(gc *gin.Context) {
 			if rows[axis][jKey][0] == "Other" {
 				return true
 			}
-			return sums[axis][iKey] > sums[axis][jKey]
+			if input.LimitType == "Max" {
+				return getMax(points[axis][iKey]) > getMax(points[axis][jKey])
+			} else {
+				return sums[axis][iKey] > sums[axis][jKey]
+			}
```

Comment:
Build `maxes` like `sums` is built (or the other way around, also use a function to get the sum).

Reply:
Done, `maxes` is constructed the same way `sums` is